src/VirtoCommerce.Platform.Web/Startup.cs

```diff
@@ -142,34 +137,13 @@ public void ConfigureServices(IServiceCollection services)
                 options.UseOpenIddict();
             });
 
-            // Enable synchronous IO if using Kestrel:
-            services.Configure<KestrelServerOptions>(options =>
-            {
-                options.AllowSynchronousIO = true;
-            });
-
-            // Enable synchronous IO if using IIS:
-            services.Configure<IISServerOptions>(options =>
-            {
-                options.AllowSynchronousIO = true;
-            });
-
-            services.Configure<CookiePolicyOptions>(options =>
-            {
-                // This lambda determines whether user consent for non-essential cookies is needed for a given request.
-                options.CheckConsentNeeded = context => true;
-                options.MinimumSameSitePolicy = SameSiteMode.None;
-            });
+            services.ConfigureWebServer();
 
             var authBuilder = services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
                                       //Add the second ApiKey auth schema to handle api_key in query string
                                       .AddScheme<ApiKeyAuthenticationOptions, ApiKeyAuthenticationHandler>(ApiKeyAuthenticationOptions.DefaultScheme, options => { })
                                       .AddCookie();
 
-            services.AddSecurityServices(options =>
```

Comment:
As I remember without this line the platform didn't work with shared token authorization https://github.com/VirtoCommerce/vc-platform/blob/master/docs/techniques/sharing-bearer-tokens-across-platform-instances.md

and also this can somehow relate to how the platform understands the  JWT token and SSO with Azure AD.
I  suggest you leave this line as is.